contribs/freight/src/main/java/org/matsim/contrib/freight/jsprit/TransportCosts.java

```diff
@@ -0,0 +1,15 @@
+package org.matsim.contrib.freight.jsprit;
+
+import com.graphhopper.jsprit.core.problem.cost.VehicleRoutingTransportCosts;
+import org.matsim.api.core.v01.network.Network;
+import org.matsim.core.router.util.LeastCostPathCalculator;
+import org.matsim.core.router.util.TravelTime;
+
+/**
+ * @author: Steffen Axer
+ */
+public interface TransportCosts extends VehicleRoutingTransportCosts {
```

Comment:
Do we have a better name for it? Please have a look to my comment in the PR conversation.